docs/en/latest/getting-started/README.md

```diff
@@ -54,7 +54,13 @@ curl "http://127.0.0.1:9080" --head | grep Server
 If everything is ok, you will get the following response:
 
 ```text
-Server: APISIX/3.1.0
+Server: APISIX/Version
+```
+
+Version is the version number of APISIX, e.g. APISIX/3.3.0
```

Comment:
Is  this better?
`Version` refers to the version of APISIX that you have installed. For example, APISIX 3.3.0.

Reply:
fixed